docs/docs/statistics/sequential.mdx

```diff
@@ -42,11 +42,11 @@ For GrowthBook, we selected a method that would work for the wide variety of exp
 
 Specifically, GrowthBook's confidence sequences, which take the place of confidence intervals, become:
 
-$$\left(\hat{\mu} \pm \hat{\sigma}^2*N * \sqrt{\frac{2(N\rho^2 + 1)}{N^2\rho^2}\log\left(\frac{\sqrt{N\rho^2 + 1}}{\alpha}\right)}\right)$$
+$$\left(\hat{\mu} \pm \hat{\sigma}*N * \sqrt{\frac{2(N\rho^2 + 1)}{N^2\rho^2}\log\left(\frac{\sqrt{N\rho^2 + 1}}{\alpha}\right)}\right)$$
 
 $$\rho = \sqrt{\frac{-2\text{log}(\alpha) + \text{log}(-2 \text{log}(\alpha) + 1)}{N^*}}$$
 
-In the above, $\hat{\mu}$ is our estimated uplift, $\hat{\sigma}^2$ is the estimated variance of the uplift, $\alpha$ is our significance level (defaults to 0.05), and $N$ is the sum of the two variation's sample sizes.
+In the above, $\hat{\mu}$ is our estimated uplift, $\hat{\sigma}$ is the estimated variance of the uplift, $\alpha$ is our significance level (defaults to 0.05), and $N$ is the sum of the two variation's sample sizes.
```

Comment:
Thanks! Shouldn't this sigma remain squared, or we should call it the standard error?

Reply:
good catch, pushed this language change